doc/sphinx/package_service_mapping.json

```diff
@@ -868,7 +896,121 @@
   "azure-mgmt-appconfiguration": {
     "category": "Management",
     "service_name": "App Configuration",
-    "manually_generated": true
+    "namespaces": [
+      "azure.mgmt.appconfiguration"
+    ]
+  },
+  "azure-mgmt-privatedns": {
+    "category": "Management",
+    "service_name": "Private DNS",
+    "namespaces": [
+      "azure.mgmt.privatedns"
+    ]
+  },
+  "azure-mgmt-frontdoor": {
+    "category": "Management",
+    "service_name": "Network",
+    "namespaces": [
+      "azure.mgmt.frontdoor"
+    ]
+  },
+  "azure-mgmt-hybridcompute": {
+    "category": "Management",
+    "service_name": "Hybrid Compute",
+    "namespaces": [
+      "azure.mgmt.hybridcompute"
+    ]
+  },
+  "azure-mgmt-healthcareapis": {
+    "category": "Management",
+    "service_name": "Healthcare APIs",
+    "namespaces": [
+      "azure.mgmt.healthcareapis"
+    ]
+  },
+  "azure-mgmt-edgegateway": {
+    "category": "Management",
+    "service_name": "Edge Gateway",
+    "namespaces": [
+      "azure.mgmt.edgegateway"
+    ]
+  },
+  "azure-mgmt-deploymentmanager": {
+    "category": "Management",
+    "service_name": "Deployment Manager",
+    "namespaces": [
+      "azure.mgmt.deploymentmanager"
+    ]
+  },
+  "azure-mgmt-peering": {
+    "category": "Management",
+    "service_name": "Peering",
+    "namespaces": [
+      "azure.mgmt.peering"
+    ]
+  },
+  "azure-mgmt-appplatform": {
+    "category": "Management",
+    "service_name": "App Platform",
```

Comment:
I see either "App Service" or "App Configuration" or "Web App fpr Containers"

Reply:
I'll dump it into App Service